File: Codes from the AI-assistants/Old Experiments/[Experimental] mandelbrot/Basic Prompt - TBR Mode/Mandelbrot-deepseek-basic-tbr.java
import java.io.BufferedWriter;
import java.io.FileWriter;
import java.io.IOException;

public class Mandelbrot {
    
    static class Complex {
        double real;
        double imag;
        
        Complex(double real, double imag) {
            this.real = real;
            this.imag = imag;
        }
        
        Complex square() {
            return new Complex(real * real - imag * imag, 2 * real * imag);
        }
        
        Complex add(Complex other) {
            return new Complex(real + other.real, imag + other.imag);
        }
        
        double magnitude() {
            return Math.sqrt(real * real + imag * imag);
        }
    }
    
    public static int mandelbrotIterations(Complex c, int maxIter) {
        Complex z = new Complex(0, 0);
        int iter = 0;
        
        while (z.magnitude() <= 2.0 && iter < maxIter) {
            z = z.square().add(c);
            iter++;
        }
        
        return iter;
    }
    
    public static void generateMandelbrotPBM(int N, int maxIter, String filename) {
        double xMin = -2.0, xMax = 1.0;
        double yMin = -1.5, yMax = 1.5;
        double xStep = (xMax - xMin) / (N - 1);
        double yStep = (yMax - yMin) / (N - 1);
        
        try (BufferedWriter writer = new BufferedWriter(new FileWriter(filename))) {
            // Write PBM header
            writer.write("P1\n");
            writer.write(N + " " + N + "\n");
            
            for (int i = 0; i < N; i++) {
                StringBuilder row = new StringBuilder();
                for (int j = 0; j < N; j++) {
                    Complex c = new Complex(xMin + j * xStep, yMin + i * yStep);
                    int iter = mandelbrotIterations(c, maxIter);
                    
                    // 1 for in set (black), 0 for out of set (white)
                    int pixel = (iter == maxIter) ? 1 : 0;
                    row.append(pixel).append(" ");
                }
                writer.write(row.toString().trim());
                writer.newLine();
            }
            
            System.out.println("Mandelbrot set saved to " + filename);
            
        } catch (IOException e) {
            System.err.println("Error writing file: " + e.getMessage());
        }
    }
    
    public static void main(String[] args) {
        int N = 800;
        int maxIter = 1000;
        
        generateMandelbrotPBM(N, maxIter, "mandelbrot.pbm");
    }
}

File: Codes from the AI-assistants/Old Experiments/[Experimental] mandelbrot/Basic Prompt - TBR Mode/Mandelbrot-deepseek-basic-tbr.py
def generate_mandelbrot_pbm(N, max_iter, filename="mandelbrot.pbm"):
    """
    Generate a Mandelbrot set and save as PBM format
    """
    # Define the complex plane region
    x_min, x_max = -2.0, 1.0
    y_min, y_max = -1.5, 1.5
    
    x_step = (x_max - x_min) / (N - 1)
    y_step = (y_max - y_min) / (N - 1)
    
    with open(filename, 'w') as f:
        # Write PBM header
        f.write(f"P1\n{N} {N}\n")
        
        for i in range(N):
            row = []
            for j in range(N):
                # Convert pixel coordinates to complex number
                c = complex(x_min + j * x_step, y_min + i * y_step)
                
                # Mandelbrot iteration
                z = 0j
                for iteration in range(max_iter):
                    z = z * z + c
                    if abs(z) > 2.0:
                        break
                
                # 1 for in set (black), 0 for out of set (white)
                pixel = 1 if iteration == max_iter - 1 else 0
                row.append(str(pixel))
            
            f.write(" ".join(row) + "\n")
    
    print(f"Mandelbrot set saved to {filename}")

# Usage
if __name__ == "__main__":
    generate_mandelbrot_pbm(800, 1000)